### PR TITLE
feat(connect): all backupDevice and recoveryDevice even for devices w…

### PR DESCRIPTION
**Pull request by mroz22**
…ith required firmware update

cc @MiroslavProchazka 

test device: `T` & `2.1.8` (first lower then `required: true`

### CODE REVIEW

File: suite-common/connect-init/src/connectInitThunks.ts [hunk 1]
```diff
@@ -156,7 +156,7 @@ export const connectInitThunk = createThunk(
                 pendingTransportEvent: selectIsPendingTransportEvent(getState()),
                 transports: selectDebugSettings(getState()).transports,
                 _sessionsBackgroundUrl,
-                // debug: true, // Enable debug logs in TrezorConnect
+                debug: true, // Enable debug logs in TrezorConnect
```
Comment (mroz22) [inline suite-common/connect-init/src/connectInitThunks.ts]: todo

File: packages/connect/src/core/AbstractMethod.ts [hunk 2]
```diff
@@ -306,8 +306,10 @@ export abstract class AbstractMethod<Name extends CallMethodPayload['method'], P
         if (!version) return;
 
         if (
-            device.firmwareStatus === 'required' ||
-            !versionUtils.isNewerOrEqual(version, range.min)
+            this.name !== 'backupDevice' &&
+            this.name !== 'recoveryDevice' &&
```
Comment (komret) [inline packages/connect/src/core/AbstractMethod.ts:311]: nit: `Array.includes`?

**Review by komret — APPROVED** [review]

File: packages/connect/src/core/AbstractMethod.ts (hunk 2 above)
Comment (peter-sanderson) [reply packages/connect/src/core/AbstractMethod.ts:311]: Doing this in type-safe way is problematic because type of `this.name` is generic
image

Comment (peter-sanderson) [reply packages/connect/src/core/AbstractMethod.ts:311]: And this works, but allows you to do a terrible mistake without TS warning you about it:

image

Comment (peter-sanderson) [reply packages/connect/src/core/AbstractMethod.ts:311]: I prefer the `includes` approach as well, but I dont think its usable here

Comment (mroz22) [reply packages/connect/src/core/AbstractMethod.ts:311]: :octocat:

**Review by peter-sanderson — APPROVED** [review]

File: suite-common/connect-init/src/connectInitThunks.ts (hunk 1 above)
Comment (mroz22) [reply suite-common/connect-init/src/connectInitThunks.ts]: should be remoevd

File: packages/suite/src/views/recovery/index.tsx [hunk 3]
```diff
@@ -179,11 +179,13 @@ export const Recovery = ({ onCancel }: ForegroundAppProps) => {
...
                 }
 
-                return device.features.capabilities.includes('Capability_PassphraseEntry') ? (
-                    <EnterOnDeviceStep />
-                ) : (
-                    <T1B1InputStep />
-                );
+                // Do not rely on Capability_PassphraseEntry feature. For ancient firmwares it's not there,
+                // and we want to allow devices that have unsupported FW to be able to check the seed.
+                if (device.features.internal_model === DeviceModelInternal.T1B1) {
```
Comment (Lemonexe) [inline packages/suite/src/views/recovery/index.tsx:184]: :+1: I mindlessly copied old code, this makes much more sense :bow:

File: packages/connect/src/core/AbstractMethod.ts (hunk 2 above)
Comment (komret) [reply packages/connect/src/core/AbstractMethod.ts:311]: Ok, that's weird, let's keep it as is, then.

File: suite-common/connect-init/src/connectInitThunks.ts (hunk 1 above)
Comment (komret) [reply suite-common/connect-init/src/connectInitThunks.ts]: yes

**Comment by bosomt** [comment]
**QA OK**

/web/backup
/web/recovery

present and working

**Pull request merged**